docs/reference/commandline/kill.md

```diff
@@ -19,3 +19,9 @@ parent = "smn_cli"
 
 The main process inside the container will be sent `SIGKILL`, or any
 signal specified with option `--signal`.
+
+> **Note:**
+> `ENTRYPOINT` and `CMD` in the *shell* form will be started as a
+> subcommand of `/bin/sh -c`, which does not pass signals. This means
+> that the executable will not be the container’s PID 1 - and will
+> not receive Unix signals.
```

Comment:
Thanks @ehaydenr! We try to avoid using future tense in the docs (not your fault really, because I see it's used in this file already), perhaps you could use something like;

```Markdown
> **Note**:
> `ENTRYPOINT` and `CMD` in the *shell* form run as a subcommand of `/bin/sh -c`,
> which does not pass signals. This means that the executable is not the container’s PID 1
> and does not receive Unix signals.
```